diesel_cli/src/print_schema.rs

```diff
@@ -248,23 +266,66 @@ pub fn output_schema(
     Ok(out)
 }
 
+#[derive(Debug, PartialEq, Clone, Eq)]
+struct CustomColumnType {
+    table_name: TableName,
+    column_name: String,
+    column_type: ColumnType,
+}
+
+impl CustomColumnType {
+    fn new(
+        backend: Backend,
+        table_name: TableName,
+        column_name: String,
+        column_type: ColumnType,
+    ) -> Self {
+        #[cfg(feature = "mysql")]
+        if backend == Backend::Mysql {
+            let name_snake = format!(
+                "{}_{}_{}",
+                &table_name.rust_name, &column_name, &column_type.rust_name
+            );
+
+            let mut column_type = column_type;
+            column_type.rust_name = name_snake
+                .split('_')
+                .map(|word| {
+                    let mut c = word.chars();
+
+                    match c.next() {
+                        None => String::new(),
+                        Some(f) => f.to_uppercase().collect::<String>() + c.as_str(),
+                    }
+                })
+                .collect();
+
```

Comment:
Just a thought: 
Wouldn't using an array of string references: `[&table_name.rust_name, &column_name, &column_type.rust_name]` be a bit more efficient? 
* No allocation of Heap memory for `String`
* Not having to `split('_')`
I made an example:
```Rust
column_type.rust_name = [&table_name.rust_name, &column_name, &column_type.rust_name]
                .iter()
                .map(|word| {
                    let mut c = word.chars();

                    match c.next() {
                        None => String::new(),
                        Some(f) => f.to_uppercase().collect::<String>() + c.as_str(),
                    }
                })
                .collect();
```
Just a thought!

Reply:
There is an even simpler method here. We can just use `heck` and call the [`to_upper_camel_case` method](https://docs.rs/heck/0.4.1/heck/trait.ToUpperCamelCase.html#tymethod.to_upper_camel_case). https://github.com/diesel-rs/diesel/pull/3530